<?xml version="1.0" encoding="UTF-8"?>
<!-- ============================================================
     OCRtoODT â€” Settings Dialog UI
     File: ui/settingsdialog.ui

     Responsibility:
         - Host all application settings panes
         - Provide tab-based navigation
         - Apply or cancel settings changes
         - All user-visible strings are English source texts
           and are translatable via Qt AUTOUIC (.ts files)
============================================================ -->
<ui version="4.0">
 <class>SettingsDialog</class>

 <widget class="QDialog" name="SettingsDialog">
  <property name="windowTitle">
   <string>OCRtoODT Settings</string>
  </property>

  <layout class="QVBoxLayout" name="verticalLayoutMain">

   <!-- ===================== Tabs ===================== -->
   <item>
    <widget class="QTabWidget" name="tabWidget"/>
   </item>

   <!-- ===================== Buttons ===================== -->
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">

     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </spacer>
     </item>

     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnResetDefaults">
       <property name="text">
        <string>Reset to Defaults</string>
       </property>
      </widget>
     </item>

     <!-- Import configuration -->
     <item>
      <widget class="QPushButton" name="btnImportConfig">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>

     <!-- Export configuration -->
     <item>
      <widget class="QPushButton" name="btnExportConfig">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>

     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>


    </layout>
   </item>

  </layout>
 </widget>
</ui>

